.nvmrc

```diff
@@ -0,0 +1 @@
+14.15.4
```

Comment:
Is this needed in a yarn project?

Reply:
.nvmrc isn't package-manager specific, but the "engines" field in package.json is, and we could set it, so I've added that! IIUC, it's probably useful to keep .nvmrc where it is, as it makes applying the right version easy w/ `nvm`, `fnm` etc.